script/Ciel/Day 14/14_01B_02_CIEL14B_4.txt

```diff
@@ -86,192 +94,213 @@ I know what this scenery is.
 }
 [sha:635ee03aae3e436959a01d1664960f15d9f7bd88]{
 -- Page 257, Offset 40180.
 -- 「ははは……は―――」
-"Hahaha... ha―――"
+"%{i}Hahaha%{/i}... %{i}ha%{/i}―――"
 }
 [sha:f0e0acfd8f717a8d9402ca64d644fea0f58ae955]{
 -- Page 257, Offset 40181.
 -- 俺が―――ロアが相手にしようとしていたものの正体を、ビジュアルとして理解した。
-I have a visual understanding of what I―――Roa was trying to go up against.
+This visual drives home just what it was that I――that Roa was trying to go up against.
 }
 [sha:c64eb21250e302666c0fd4f49591aa50c8492fe1]{
 -- Page 257, Offset 40182.
 -- アイツが巨人なのは当然だ。
-Of course she would be massive.
+It's natural that she'd be a giant.
 }
 [sha:db1a26279de6738bda5aef250da3b1bcbe9652fa]{
 -- Page 257, Offset 40183.
 -- <人|お><間|れ>たちが虫の如き存在なのも当然だ。
-The fact that we exist as insects in this plane is natural.
+It's natural that we humans would be nothing but ants.
 }
 [sha:a8697e7d5c92147a3a7ee47b31869d896bb24652]{
 -- Page 258, Offset 40184.
 -- 異様なまでの湿度と高温。
-An unusual level of humidity and high temperatures.
+This place is unnaturally hot and humid.
 }
 [sha:fc09c1d3be8cb873074b8f99b02f7af64b211e63]{
 -- Page 258, Offset 40185.
 -- むせかえる原始の色彩。
-A suffocating primordial color.
+Suffocated by primordial colors.
 }
 [sha:54694e0f4d72ba0b0346be93b3f6def47d10ac2c]{
 -- Page 258, Offset 40186.
 -- <視界|せかい>を覆う、夥しいまでの緑のヴェール。
-A vast number of green veils covering the world.
+Countless green veils drape themselves across the world and obscure my vision.
 }
 [sha:af8aff5abaa7b9c73b4f15a81fca31424b8beec6]{
 -- Page 258, Offset 40187.
 -- 酸素は薄く、気体は生命を殺すほどの濃度となって充満している。
-The oxygen is thin, and the gases are so thick to the point where it can kill life.
+Oxygen is scarce, and the atmosphere has grown so thick as to be poisonous to life.
 }
 [sha:6a286e3680764bc1a6e05dac1a9b43b25420862c]{
 -- Page 259, Offset 40188.
 -- 人間の知恵を嘲笑う、獰猛な自然のカタチ。
-A vicious form of nature that makes a mockery out of human wisdom.
+A savage form of nature that makes a mockery of human wisdom.
 }
 [sha:4d2f9c4352e5882f7d3ae65a97f6018bb72f18d1]{
 -- Page 259, Offset 40189.
 -- そこはもうビル街ではなかった。
-This could no longer be called a business district.
+This place isn't an office district anymore.
 }
 [sha:cab28ba514ef827aee7bfa3786de4352d80745ee]{
 -- Page 259, Offset 40190.
 -- 自然に侵食され、廃墟になった文明の跡地でもなかった。
-It couldn't be called a civilization that turned to ruin after being encroached by nature either.
+It isn't the site of a fallen civilization that has been overtaken by nature, either.
 }
 [sha:9fefe32a9442d4a69565a2ef2627dfd1cfcaa198]{
 -- Page 259, Offset 40191.
 -- もっともっと過去の世界。
-This is a world from a much earlier past.
+This is a world from the distant past.
 }
 [sha:04bec1078a5ce34ed5bd070973ac76d88888b2fd]{
 -- Page 259, Offset 40192.
 -- 人類が発生する前―――何億年も昔の、太古の地球の姿だった。
-An ancient world from hundreds of millions of years past―――long before the existence of mankind.
+An ancient vision of earth from hundreds of millions of years ago――long before mankind came into existence.
```

Comment:
Hmm kinda prefer current. "Ancient vision of X" is definitely something I've read before.

Reply:
Just realized mine repeats 'past'... so that's bad

I'm sort of also hesitant with 'vision' because it can imply just a glimpse, essentially, when it's more like... real as heck
I've feel like I've heard it too, but in other contexts. Google doesn't elucidate any too relevant examples, sadly. They're mostly about yoga or something

>It is how the earth looked in ancient times, hundreds of millions of years ago——